File: 1323-maximum-69-number/1323-maximum-69-number.cpp
class Solution {
public:
    int maximum69Number (int num) {
        int lt = log10(num);
        int temp = num;
        while(temp > 0){
            int du = temp / (int)pow(10,lt);
            if(du == 6){
                num+= 3*pow(10,lt);
                break;
            }
            temp = temp % (int)pow(10,lt);
            lt--;
        }
        return num;
    }
};

File: 1323-maximum-69-number/1323-maximum-69-number.py
class Solution(object):
    def maximum69Number (self, num):
        fnum = list(str(num))
        for i in range(0, len(fnum)):
            if fnum[i] == '6':
                fnum[i] = '9'
                break
        return (int)("".join(fnum))
        